### ISSUE TITLE
Arrow at end of file.

### ISSUE THREAD

**Issue by braddunbar** [clutchski/coffeelint#148]
Using the `arrow_spacing` option:

``` sh
$ printf 'class A\n  f: ->' | coffeelint -s -f coffeelint.json
  ✗ stdin
     ✗ #2: Function arrow (->) must be spaced properly.

✗ Lint! » 1 error and 0 warnings in 1 file
```

Expected this to be valid, but it is not.
